# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/Http3Session.h
# Commit: ee9744982673
# Full Hash: ee974498267381418f06efd6ab92c1f2c5f3735d
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-15 16:44:51
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r f5a4bfdaba40 -r ee9744982673 netwerk/protocol/http/Http3Session.h
--- a/netwerk/protocol/http/Http3Session.h	Tue Nov 15 05:34:40 2022 +0000
+++ b/netwerk/protocol/http/Http3Session.h	Tue Nov 15 05:34:40 2022 +0000
@@ -7,17 +7,18 @@
 #ifndef Http3Session_H__
 #define Http3Session_H__
 
+#include "HttpTrafficAnalyzer.h"
+#include "mozilla/UniquePtr.h"
+#include "mozilla/WeakPtr.h"
+#include "mozilla/net/NeqoHttp3Conn.h"
+#include "nsAHttpConnection.h"
+#include "nsDeque.h"
 #include "nsISupportsImpl.h"
 #include "nsITimer.h"
 #include "nsIUDPSocket.h"
-#include "mozilla/net/NeqoHttp3Conn.h"
-#include "nsAHttpConnection.h"
 #include "nsRefPtrHashtable.h"
+#include "nsTHashMap.h"
 #include "nsWeakReference.h"
-#include "HttpTrafficAnalyzer.h"
-#include "mozilla/UniquePtr.h"
-#include "mozilla/WeakPtr.h"
-#include "nsDeque.h"
 
 /*
  * WebTransport